### ISSUE TITLE
page break

### ISSUE THREAD

**Issue by OtharSmirnow** [jdum/odfdo#9]
hi,
I can't find the way to insert a page break.
Is there already a function for this as paragraph does not accept 
doc.body.append(Paragraph("\u000C"))
https://www.fileformat.info/info/unicode/char/000c/index.htm

**Comment by jdum** [comment]
hi, 
maybe the tag: <text:soft-page-break>, see http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#element-text_soft-page-break
however not implemented as keyword (because very uncommon), so use with:
Element.from_tag("<text:soft-page-break>").

Or what you need is probably:
- close the paragraph
- open a new paragraph with some dedicated style that contains <style:paragraph-properties fo:break-before="page"/>
so basically, it is a paragraph style, then better to copy style from other document because styles not easy to write by hand.

sorry, there is no simple answer to that question.

**Comment by OtharSmirnow** [comment]
document.body.append(Paragraph('', style='breakPageAfter'))

style defined in style_document

did the trick Thank you
